From 9c45ac7c254004e4041fa906cc5b14ec2013d4bd Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 18 Apr 2025 21:11:05 -0500
Subject: [PATCH] Switch to ament_cmake_ros_core package

---
 CMakeLists.txt | 3 ++-
 package.xml    | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7612d1c..abf7c8a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,7 +11,8 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic -pthread)
 endif()
 
-find_package(ament_cmake_ros REQUIRED)
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_ros_core REQUIRED)
 
 find_package(rmw REQUIRED)
 find_package(rmw_dds_common REQUIRED)
diff --git a/package.xml b/package.xml
index a5573bb..5120361 100644
--- a/package.xml
+++ b/package.xml
@@ -8,7 +8,8 @@
   <license>GPLv3</license>
 
   <buildtool_depend>rosidl_cmake</buildtool_depend>
-  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_ros_core</buildtool_depend>
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
   <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
-- 
2.43.0

